readme.md

```diff
@@ -446,6 +446,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Fuzzing(for software security) related papers and their implementation codes, tools reported at the top security conference.
```

Comment:
This still doesn't explain what **fuzzing** is.

Reply:
I added a sentence describing what fuzzing is, (a kind of software security test method).
More details are already included in the link.